workbench/src/renderer/styles/style.css

```diff
@@ -300,29 +308,35 @@ body {
 	border-radius: 0;
 }
 
-.sidebar-setup {
+.sidebar-row {
+	display: flex;
+	flex-direction: column;
+	font-size: 1.0rem;
+	padding-left: 1.0rem;
```

Comment:
Is the `.0` needed?

Reply:
No. Lots of times a `rem` value will not be a whole number though.